setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 requirements = [
-    "flask>=1.0.0",
+    "flask>=2.0.0",
```

Comment:
Good catch. I updated this in #150 as well. We need flask 2.1.0 for compatibility with newer werkzeug versions, which I also pinned. The "oldest" requirements job that I added in #150 caught this for us. Please take the changes from #150 instead of these changes.